lib/player.rb

```diff
@@ -0,0 +1,56 @@
+require_relative 'scoring'
+require_relative 'tile_bag'
+
+module Scrabble
+  class Player
+    attr_reader :name, :plays, :tiles
+    def initialize(name)
+      @name = name
+      @plays = []
+      @tiles = []
+      @tile_bag = TileBag.new
+    end
+
+    def won?
+      false unless total_score > 100
```

Comment:
Does this actually work? It looks like it will return either `false` or `nil`, which are both falsey.